RELEASE_NOTES.md

```diff
@@ -16,17 +16,25 @@ CSM 1.3 contains approximately 500 changes spanning bug fixes, new feature devel
 
 * Integrated Kyverno Native Policy Management engine
 * Ansible has been added to NCNs
-* Added support for the Replace/Remove/Add NCN procedures
+* Added support for procedures:
+  * Replace/Remove/Add NCNs
+  * Add River cabinets
 * Integrated Argo Server workflow engine for Kubernetes
 * Technology Preview: BOS V2 (Boot Orchestration) Asynchronous boot state handling and CRUS replacement for rolling upgrade
 * Technology Preview: Tenant and Partition Management Service (TAPMS)
-* Support for setting of Bios Settings through SCSD
-* Ability to set power cap on n number of computes
+* Added support for using SCSD to enable or disable TPM BIOS setting on Gigabyte and HPE hardware
+* Boot NCNs using private S3 bucket
+* Enable IMS recipe templating to allow for dynamic repository selection
+* CSM health check performance improvements
+  * HMS tests now execute in parallel using Helm Test
+  * NCN and Kubernetes health checks now execute in parallel and eliminate lengthy output for tests that pass
 
 ### New hardware support
 
-* Olympus Antero Blade (AMD Genoa) with Slingshot 11
 * Aruba JL705C, JL706C, JL707C management network switches
+* Milan-based DL325 as a Compute Node
+* Olympus Antero Blade (AMD Genoa) with Slingshot 11
+  * No power capping support
```

Comment:
@nieuwsma do we need to mention something about the NID allocation issue for this blade type?

Reply:
Um... I dont know. 
I think thats probably something else we handle later.